tests/spread/general/build_environment/snaps/build-environment-test/snap/snapcraft.yaml

```diff
@@ -0,0 +1,29 @@
+name: build-environment-test
+version: '1.0'
+summary: Quick snap to ensure build-environment works
+description: Quick snap to ensure build-environment works
+
+base: core18
+grade: devel
+confinement: strict
+
+apps:
+  build-environment-test:
+    command: echo "$FOO"
```

Comment:
wait, how is this possible?

Reply:
Oh it's not! At least, I hope it's not :grin: . This is here so we can [make _sure_ it's not](https://github.com/snapcore/snapcraft/pull/2322/files#diff-eb40a183cf74c2ef453f34aa1ebe4852R27).